operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -104,9 +104,14 @@ type CentralComponentSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="User-facing TLS certificate secret",order=3
 	DefaultTLSSecret *LocalSecretReference `json:"defaultTLSSecret,omitempty"`
 
+	// Configures monitoring endpoint for central. Monitoring endpoint allows
+	// other services, like Prometheus, to collect metrics provided by central.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=4
```

Comment:
Just as a suggestion from my side, in which direction we should move with the comment, let's see what other's are suggesting:

```suggestion
	// Configures monitoring for Central. 
	// Central provides metrics in prometheus compatible format.
	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=4
```

Reply:
I actually like the version which mentions "endpoint" more. Seeing just "configures monitoring", some might think that this provides a full solution, including a graphana dashboard ;-)

Reply:
I have tweaked this a bit. I like the statement `Prometheus compatible format`, so I have integrated that part.